src/components/deploymentGroups/DeploymentGroupList.tsx

```diff
@@ -303,17 +303,19 @@ export default class DeploymentGroupList extends Component<BulkActionListProps,
         this.props.history.push(LINK)
     }
 
+    renderCreateButton = () => {
+        return (
+            <button type="button" className="flex cta h-32 lh-n" onClick={() => this.redirectToCreateGroup()}>
+                <Add className="icon-dim-20" />
+                Create Group
+            </button>
+        )
+    }
+
     render() {
         return (
             <div>
-                <PageHeader
-                    headerName="Deployment Groups"
-                    showCreateButton={this.state.view === ViewType.FORM ? true : false}
-                    buttonText="group"
-                    onClickCreateButton={this.redirectToCreateGroup}
-                    CreateButtonIcon={Add}
-                    showIconBeforeText={true}
-                />
+                <PageHeader headerName="Deployment Groups" />
```

Comment:
did we forget to add renderCreateButton here?

Reply:
fixed